scripts/benchmarks/pyproject.toml

```diff
@@ -6,70 +6,31 @@ authors = ["Charles Marsh <charlie.r.marsh@gmail.com>"]
 
 [tool.poetry.dependencies]
 python = ">=3.10,<3.12"
-autoflake = "^2.0.0"
-flake8 = "^6.0.0"
-pycodestyle = "^2.10.0"
-pyflakes = "^3.0.1"
-pylint = "^2.15.10"
-black = "^22.12.0"
-isort = "^5.11.4"
-flake8-2020 = { version = "*", optional = true }
-flake8-annotations = { version = "*", optional = true }
-flake8-bandit = { version = "*", optional = true }
-flake8-blind-except = { version = "*", optional = true }
-# flake8-boolean-trap = { version = "*", optional = true }
-flake8-bugbear = { version = "*", optional = true }
-flake8-builtins = { version = "*", optional = true }
-flake8-commas = { version = "*", optional = true }
-flake8-comprehensions = { version = "*", optional = true }
-flake8-datetimez = { version = "*", optional = true }
-flake8-debugger = { version = "*", optional = true }
-flake8-docstrings = { version = "*", optional = true }
-# flake8-eradicate = { version = "*", optional = true }
-flake8-errmsg = { version = "*", optional = true }
-flake8-implicit-str-concat = { version = "*", optional = true }
-# flake8-import-conventions = { version = "*", optional = true }
-flake8-isort = { version = "*", optional = true }
-flake8-pie = { version = "*", optional = true }
-flake8-print = { version = "*", optional = true }
-flake8-quotes = { version = "*", optional = true }
-flake8-return = { version = "*", optional = true }
-flake8-simplify = { version = "*", optional = true }
-flake8-super = { version = "*", optional = true }
-flake8-tidy-imports = { version = "*", optional = true }
-pandas-vet = { version = "*", optional = true }
-pep8-naming = { version = "*", optional = true }
+autoflake = "*"
+autopep8 = "*"
+black = "*"
+flake8 = "*"
+isort = "*"
+pycodestyle = "*"
+pyflakes = "*"
+pylint = "*"
+yapf = "*"
```

Comment:
Honestly don't know if Poetry is worth preserving here. I end up using `pipx` for these selectively.